scripts/postpublish.js

```diff
@@ -8,5 +8,17 @@ const betaExpr = /\d+\.\d+\.\d+-.*/
 const legacyExpr = /0\.\d+\.\d+/
 
 if (!betaExpr.test(pkg.version) && !legacyExpr.test(pkg.version)) {
-  exec(`node scripts/publish_docs.js "v${pkg.version}"`)
+  const releaseBranches = exec.pipe('git branch | grep -E "v\\d+\\.x"')
```

Comment:
This depends on all the release branches (or at least the latest one) being available locally, which might not be the case.

You'll need to fetch origin first, then use `git branch -a`

Reply:
Done.